client/app/kubernetes/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,77 @@
+    <translation id="kube_nodes_add_flavor_family_accelerated">GPU</translation>
+    <translation id="kube_nodes_add_flavor_family_vps">SANDBOX</translation>
+    <translation id="kube_nodes_add_flavor">Choisissez un modèle d'instance dans le catalogue OVH public cloud</translation>
+    <translation id="kube_nodes_add_pick_flavor">Choisir un modèle d'instance</translation>
+    <translation id="kube_nodes_add_error">Une erreur est survenue lors de l'ajout du nœud : {{ message }}</translation>
+    <translation id="kube_nodes_add">Ajouter un nœud</translation>
+
+    <translation id="kube_nodes_delete">Supprimer le nœud</translation>
+    <translation id="kube_nodes_delete_confirmation">Êtes-vous sûr de supprimer {{ node }} ?</translation>
+    <translation id="kube_nodes_delete_warning">Cette action est irréversible.</translation>
+    <translation id="kube_nodes_delete_enter">Entrez "DELETE" dans le champ ci-dessous pour confirmer</translation>
+    <translation id="kube_nodes_delete_success">Le nœud est en cours de suppression</translation>
+    <translation id="kube_nodes_delete_error">Une erreur est survenue lors de la suppression du nœud : {{ message }}</translation>
+
+    <translation id="kube_nodes_node">Nœud</translation>
+    <translation id="kube_nodes_error">Une erreur est survenue lors du chargement des nœuds</translation>
+    <translation id="kube_nodes_refresh">Recharger le tableau </translation>
+    <translation id="kube_nodes_flavor">Modèle d'instance</translation>
+    <translation id="kube_nodes_flavor_error">Une erreur est survenue lors du chargement du type d'instance associé</translation>
+    <translation id="kube_nodes_project_error">Une erreur s'est produite lors du chargement du projet associé</translation>
+    <translation id="kube_nodes_description_project_1">Votre cluster Kubernetes utilise des nœuds pour héberger vos conteneurs et services. Tous vos nœuds seront lancés dans le projet public cloud</translation>
+    <translation id="kube_nodes_description_project_2">(nous permettrons de changer cela dans le futur). Ces instances vous seront facturées normalement dans votre projet public cloud.</translation>
+
+    <translation id="kube_flavor">{{ name }} ({{ cpuNumber }} CPU {{ ramCapacity }}Go RAM {{ diskCapacity }}Go Disque)</translation>
+
+    <translation id="kube_containers_and_services">Conteneurs et services</translation>
+    <translation id="kube_containers_coming_soon">Bientôt disponible</translation>
+    <translation id="kube_containers_description">Vous pourrez bientôt retrouver ici tous les conteneurs et services hébergés par votre cluster Kubernetes.</translation>
+    <translation id="kube_containers_description_dashboard">Nous vous invitons à utiliser l'outil en ligne de commande "kubectl" ou installer le Daskboard Kubernetes. Il s'agit d'une interface utilisateur web maintenue par la communauté Kubernetes, elle-même exposée en tant que service au sein de votre cluster. Suivez</translation>
```

Comment:
Does `Suivez` could be move to the next translation key `kube_containers_description_dashboard_link`?
It seems more convenient from a translator point a view.